src/actions/Entities/NameNoteActions.js

```diff
@@ -103,11 +106,22 @@ const saveEditing = optionalNameID => (dispatch, getState) => {
   dispatch(reset());
 };
 
+const createNotes = (nameNotes = []) => {
+  UIDatabase.write(() => {
+    nameNotes.forEach(nameNote => {
+      UIDatabase.update('NameNote', nameNote);
```

Comment:
I had to do this for it to save notes:
```suggestion
      const { patientEventID, nameID } = nameNote;
      const name = UIDatabase.get('Name', nameID);
      const patientEvent = UIDatabase.get('PatientEvent', patientEventID);

      if (name && patientEvent) {
        const toSave = {
          id: nameNote.id,
          patientEvent,
          name,
          _data: JSON.stringify(nameNote?.data),
          entryDate: new Date(nameNote?.entryDate),
        };

        UIDatabase.update('NameNote', toSave);
      }
```

Reply:
Thanks!